src/pages/components/tile/usage.mdx

```diff
@@ -61,13 +63,25 @@ ecosystem below:
 ### When to use
 
 Tiles are reusable components that provide shortcuts to building cards and other
-modules. Here are some common use cases for when to use tiles:
+modules. Use tiles as part of the user's main workstream. Here are some common
+use cases for when to use tiles:
 
 - To contain related groupings of information or actions
 - To guide users to take actions or navigate
 - To present options for single or multiple selections
 - To hide or show large amounts of content
 
+### When not to use
+
+Tiles reside in the same plane as the background layer—they do not have
+elevation. Tiles organize essential information and have the same visual
+hierarchy as content within the same page.
+
+Do not add drop shadow to tiles and use them on screen to reveal secondary
+information, actions, or notifications. Use [modals](/components/modal/usage),
+[popover](/components/popover/usage), and [diaglog](/components/dialog/usage)
```

Comment:
```suggestion
[popover](/components/popover/usage), and [dialog](/components/dialog/usage)
```

Reply:
[/components/dialog/usage](https://carbondesignsystem.com/components/dialog/usage) doesn't seem to be a link, I'm getting 404